chaoslib/litmus/pod_delete/pod_failure_by_litmus.yml

```diff
@@ -60,6 +60,14 @@
         executable: /bin/bash
       register: helper_pod
 
+    - name: "[Status]: Checking the running status of the helper pod"
+      shell: >
+        kubectl get pods -l name=pod-delete-{{ run_id }} -n {{ c_ns }} -o custom-columns=:.status.phase --no-headers
+      register: pod_result
+      until: "((pod_result.stdout.split()|unique)|length) == 1 and 'Running' in pod_result.stdout"
```

Comment:
Do we need the first check there will be only one helper pod?

Reply:
As we are appending the run_id as a suffix in the pod name so there will be a single pod only. but in some cases (lack of resources or any other) the pod will try to create multiple instances sometimes. So I think it is not overhead to add this check as well. What do u think?
cc: @ksatchit